torchok/data/datasets/classification/classification.py

```diff
@@ -122,6 +125,11 @@ def get_raw(self, idx: int) -> dict:
 
         return sample
 
+    def get_sampler_weights(self):
```

Comment:
Sampler is different entity, we need split dataset entity and sampler entity. This function can't be here. Move it to Sampler class.